web/ui/react-app/src/pages/targets/SeriesLabels.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Tooltip } from 'reactstrap';
+import SeriesLabel from './SeriesLabel';
+
+interface Labels {
+  [key: string]: string;
+}
+
+export interface SeriesLabelsProps {
+  discoveredLabels: Labels;
+  labels: Labels;
+}
+
+const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
+
+const SeriesLabels: React.FC<SeriesLabelsProps> = ({ discoveredLabels, labels }: SeriesLabelsProps) => {
+  const [tooltipOpen, setTooltipOpen] = React.useState(false);
+
+  const toggle = (): void => setTooltipOpen(!tooltipOpen);
+
+  return (
+    <>
+      <div id="series-labels" className="series-labels-container">
```

Comment:
is id needed? I found such a css class but not an id

Reply:
A constant `id` definitely seems problematic for a repeated element. I bet this is why all the tooltips show up when hovering over the first target's labels.

Reply:
yes, for the Tooltip i think

Reply:
Yeah, but then the `id` has to vary between targets :)

Reply:
I knew that we didn't have to delete the idGen function :D

Reply:
ahhh i see now

Reply:
fixed